cmd/event-add.go

```diff
@@ -65,7 +69,9 @@ EXAMPLES:
 
    2. Enable bucket notification with filters parameters
      $ {{.HelpName}} s3/mybucket arn:aws:sqs:us-west-2:444455556666:your-queue --event put,delete,get --prefix photos/ --suffix .jpg
-
+   
+   3. Ignore duplicate bucket notification with --ignore-existing/-p flag
```

Comment:
Use the short form as referenced in the command line @sinhaashish we don't need to talk about `--ignore-existing`

Reply:
done.